src/telliot_feeds/reporters/tellorflex.py

```diff
@@ -98,6 +99,12 @@ async def fetch_gas_price(self, speed: str = "safeLow") -> Optional[int]:
             return None
         return int(prices[speed])
 
+    async def in_dispute(self, stake_amount: Any) -> bool:
```

Comment:
Change the var name to `new_stake_amount` or something, just so things are a bit clearer